src/containers/sidebar/sidebar.component.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Navbar from "containers/navbar";
+import { Route, Link, BrowserRouter as Router } from "react-router-dom";
+import { useSelector } from "react-redux";
+import { auth } from "database";
+const Sidebar = (props) => {
```

Comment:
Highly advice always destructuring the props here before using them in the component.

This way you avoid constantly writing `props.something` and can just write `something`

Reply:
Theres only one props on this one, I should do it for Posts (where theres lots)? or all?